doc/README.md

```diff
@@ -189,8 +192,46 @@ paths with the highest cum weight.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+* **_flat_ value**: the value of a location itself, is indicated by the box size.
+
+    *   Nodes with larger box have larger _flat_ values.
+    *   Nodes with smaller box have smaller _flat_ values.
+
+*  **_cum_ value**: the value of a location plus all of its descendants, is indicated by the nodes' color.
+
+    *   Redder nodes have greater _cum_ values.
+    *   Greyer nodes have smaller _cum_ values.
+
+*   **Dashed Edges**: some locations between the two connected locations were
+    removed.
+
+*   **Solid Edges**: one location directly calls the other.
+
+*   **Thicker & Redder Edges**: more resources (i.e. larger value) were used along that path.
+
+*   **Thinner & Greyer Edges**: less resources (i.e. smaller value) were used along that path.
+
+Let's consider the following example graph:
+
+![callgraph](images/callgraph.svg)
+
+* In terms of nodes, we notice that:
+  * `NewWriter` has a small _flat_ value and a small _cum_ value because the node is small and grey,
+  * `deflate` has a large _flat_ value and a large _cum_ value because the node is large and red,
+  * `Flush` has a small _flat_ value and a large _cum_ value because the node is small and red.
 
+* In terms of edges, we take a look at
+  * the edge between `(*Writer) Write` and `(*compressor) write`: 
+    * since it is a dashed edge, some nodes were removed between those two;
```

Comment:
Nit: these bullets use

```
* some phrase;
```

style and bullets above seem to use

```
* Full sentence.
```

style. Should we make the style consistent?

Reply:
I was trying really hard to make the whole "paragraph" a sentence. But it does seem like no matter grammatically or consistency-wise, there is no way to fit them into a single sentence. I removed every punctuation at the end of each bullet point to be consistent.